controllers/teams.js

```diff
@@ -0,0 +1,14 @@
+const teams = require('../teams')
+
+const getAllTeams = (request, response) => {
+  return response.send(teams)
+}
+
+const getTeamsById = (request, response) => {
+  const { id } = request.params
+  const locateTeam = teams.filter(team => team.id === Number(id))
```

Comment:
Find would be a better choice here since an ID will only ever be for one team and filter returns an array